integration_test/Utils.ts

```diff
@@ -0,0 +1,411 @@
+    return await this.auxiliaryWeb3.eth.sendTransaction(
+      {
+        from: this.auxiliaryFunder,
+        to: beneficiary,
+        value: web3Utils.toWei(web3Utils.toBN(2)),
+      },
+    );
+  }
+
+  public async getOriginOrganizationInstance(): Promise<Organization> {
+    const organizationAddress = await this.getOrganizationFromOSTComposer();
+
+    return interacts.getOrganization(this.originWeb3, organizationAddress);
+  }
+
+  /**
+   * It whitelists address of an account.
+   * @param worker Address to be whitelisted.
+   * @param expirationHeight Block number at which address becomes invalid.
+   * @returns Receipt object.
+   */
+  public async whitelistOriginWorker(worker: string, expirationHeight: number): Promise<any> {
+    const organizationContractInstance = await this.getOriginOrganizationInstance();
+
+    const owner = await organizationContractInstance.methods.owner().call();
+
+    const setWorkerRawTx: TransactionObject<void> = await organizationContractInstance.methods.setWorker(
+      worker,
+      expirationHeight,
+    );
+
+    const setWorkerReceipt = await SrcUtils.sendTransaction(
+      setWorkerRawTx,
+      {
+        from: owner,
+        gasPrice: '0x174876E800',
+      },
+      this.originWeb3,
+    );
+
+    return setWorkerReceipt;
+  }
+
+  public getAuxContractInstance(contractName: string, contractAddress: string) {
+    const abi = fs.readFileSync(path.join(ABIDirectoryPath, contractName.concat('.json')), 'utf8');
+    return new this.auxiliaryWeb3.eth.Contract(
+      JSON.parse(abi),
+      contractAddress,
+    );
+  }
+
+  /**
+   * It provide organization contract used in OSTComposer.
+   * @returns Organization contract address.
+   */
+  public async getOrganizationFromOSTComposer(): Promise<string> {
+    const ostComposerInstance = interacts.getOSTComposer(this.originWeb3, this.ostComposer);
+    const organizationAddress = await ostComposerInstance.methods.organization().call();
+
+    return organizationAddress;
+  }
+
+  /**
+   * It anchors state root to auxiliary chain's Anchor contract.
+   */
+  public async anchorOrigin(auxChainId: number): Promise<void> {
+    const organizationInstance = interacts.getOrganization(this.auxiliaryWeb3, mosaicConfig.auxiliaryChains[auxChainId].contractAddresses.auxiliary.anchorOrganizationAddress);
+
+    const anchorInstance = interacts.getAnchor(this.auxiliaryWeb3, mosaicConfig.auxiliaryChains[auxChainId].contractAddresses.auxiliary.anchorAddress);
+
+    const currentBlock = await this.originWeb3.eth.getBlock('latest');
+
+    const owner = await organizationInstance.methods.owner().call();
+
+    const anchorStateRootRawTx: TransactionObject<boolean> = await anchorInstance.methods.anchorStateRoot(
+      currentBlock.number,
+      currentBlock.stateRoot,
+    );
+
+    await SrcUtils.sendTransaction(
+      anchorStateRootRawTx,
+      {
+        from: owner,
+        gasPrice: '0x174876E800',
```

Comment:
Hardcode?

Reply:
Query the node to retrive the gas price.